cmd/utils/flags.go

```diff
@@ -2004,6 +2021,21 @@ func SetEthConfig(ctx *cli.Context, stack *node.Node, cfg *ethconfig.Config) {
 			cfg.NetworkId = genesis.Config.ChainID.Uint64()
 		}
 		cfg.Genesis = genesis
+	case ctx.Bool(OpBNBMainnetFlag.Name):
+		if !ctx.IsSet(NetworkIdFlag.Name) {
+			cfg.NetworkId = 204
+		}
+		cfg.Genesis = MakeGenesis(ctx)
+	case ctx.Bool(OpBNBTestnetFlag.Name):
+		if !ctx.IsSet(NetworkIdFlag.Name) {
+			cfg.NetworkId = 5611
+		}
+		cfg.Genesis = MakeGenesis(ctx)
+	case ctx.Bool(OpBNBDevnetFlag.Name):
+		if !ctx.IsSet(NetworkIdFlag.Name) {
+			cfg.NetworkId = 1322
```

Comment:
Our devnet's chainId is 901, I'm not sure about the origin of 1322, can you confirm it again?

Reply:
This refers to our qa environment. The hardcode devnet is just for the convenience of testing this mode. The qa environment also often needs to reset and change the chain config information and genesis, so the devnet information is not very important and variable. We can just change it for verification when needed one time.

Reply:
Shouldn't the configuration of the QA environment be called QANet? I ask this question because it could be confused with the DevNet of the OPBNB project. To avoid confusion for users, I suggest changing it to QANet.

Reply:
Good suggetsion. Resovled and please review again.